pages/establishment/licence-fees/personal/formatters/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import format from 'date-fns/format';
+import { Link } from '@asl/components';
+import { dateFormat } from '../../../../../constants';
+
+export default {
+  licenceHolder: {
+    format: (val, pil) => `${pil.profile.firstName} ${pil.profile.lastName}`
+  },
+  licenceNumber: {
+    format: (val, pil) => {
+      if (!pil.profile.establishments.length) {
+        return val;
+      }
+      return <Link page="pil.read" pilId={pil.id} profileId={pil.profile.id} label={val} establishmentId={pil.establishmentId} />;
```

Comment:
Are we linking to establishments that the current user might not be related to, or are they stripped elsewhere?

Reply:
It is stripped elsewhere, but it's redundant here, so I've removed it anyway.